src/repository/screens/repository.screen.js

```diff
@@ -95,12 +104,14 @@ class Repository extends Component {
   handlePress = index => {
     const {
       starred,
+      subscribed,
```

Comment:
I very very very much agree that we should improve this, not the cleanest way to do things to be honest _especially_ when adding conditional items like `fork`